app/views/users/show.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Here is detail of a specific user</h1>
\ No newline at end of file
+<%= render "shared/user", user: @user  %>
+ <div class="row m-lg-4 m-md-3 m-lg-2 m-1 justify-content-center align-items-center ">
+  <div class="col-lg-8 col-md-11 col-11 border p-lg-2 p-1">
+    <h5>Bio</h5>
+    <p><%= @user.bio %></p>
+  </div>
+  <%if @user.posts.any? %>
+    <% @user.posts.each_with_index do |post, index| %>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use the `recent_posts` method here to render only the 3 most recent posts for the user.
```suggestion
  <%if @user.posts.any? %>
    <% @user.recent_posts.each_with_index do |post, index| %>
```